.github/workflows/fixtures_feature.yaml

```diff
@@ -7,20 +7,34 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
+  feature-names:
     runs-on: ubuntu-latest
+    outputs:
+      names: ${{ steps.feature-name.outputs.names }}
     steps:
       - uses: actions/checkout@v4
         with:
-          submodules: true
-      - name: Get feature name
+          submodules: false
+      - name: Get feature names
         id: feature-name
         shell: bash
         run: |
-          echo name=${GITHUB_REF_NAME//@*/} >> "$GITHUB_OUTPUT"
+          names=$(grep -Po "^${GITHUB_REF_NAME//@*/}[^:]*" configs/feature.yaml | jq  --raw-input .  | jq -c --slurp .)
+          echo names=${names}
```

Comment:
```suggestion
```
maybe this is obsolete?

Reply:
Yeah, it's for debugging, but maybe if we incorrectly tag something at some point this could be useful?

Reply:
Ahh yeah lets keep it :)